sdk/eventgrid/Azure.Messaging.EventGrid/README.md

```diff
@@ -182,7 +182,7 @@ foreach (CloudEvent cloudEvent in cloudEvents)
             break;
         case "MyApp.Models.CustomEventType":
             // One can also specify a custom ObjectSerializer as needed to deserialize the payload correctly
-            TestPayload testPayload = await cloudEvent.GetDataAsync<TestPayload>(myCustomSerializer);
+            TestPayload testPayload = cloudEvent.GetData<TestPayload>();
```

Comment:
I don't think this code sample is right anymore, since you no longer pass in a custom object serializer?  I'm not actually sure how this works anymore with the new design

Reply:
Good catch - this probably would fail but it is a live-only test so it wasn't caught locally.